packages/ui/src/card.tsx

```diff
@@ -20,25 +20,22 @@ const CardHeader = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDiv
 );
```

Comment:
I updated this based on the update from shadcn (see changelog at bottom)
https://ui.shadcn.com/docs/components/card

This no longer assumes the card title will be an `h3`, so I updated some other parts accordingly